CHANGELOG.md

```diff
@@ -5,4 +5,4 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [3.0.1](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/compare/3.0.0...3.0.1) - 2016-05-22
 ### Fixed
 - Added missing default sort by primary key(s).
-- Allow non integer/string types as identifier (ex. uuid).
+- Allow non integer/string types as identifier (ex. uuid).
```

Comment:
It is a whitespace that is removed at end of line (see @OskarStark 's screenshot)